src/tools/genpolicy/src/policy.rs

```diff
@@ -78,21 +78,24 @@ pub struct KataSpec {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct KataProcess {
+    #[serde(default)]
     pub Terminal: bool,
+    #[serde(default)]
     pub User: KataUser,
-
+    
```

Comment:
let's not introduce whitespaces, tabs or any unnecessary stuff. best to configure VS code to truncate such things automatically.